### PR TITLE
Removed unnecessary duplicated words.

### PR DESCRIPTION
**Pull request by sanganinamrata**
## Purpose of this pull request

This pull request is to remove unnecessary duplicate words in a couple of dev-doc pages.

### CODE REVIEW

**Comment by devops-devdocs** [comment]
An admin must run tests on this PR before it can be merged.

**Comment by keharper** [comment]
@sanganinamrata Your PR includes several files in the `.idea` directory. Please remove them.

File: .idea/devdocs.iml [hunk 1]
```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
[the remark this answers is not in the document]
Comment (sanganinamrata) [reply .idea/devdocs.iml]: Done

File: _data/codebase/mftf/action-groups.yml [hunk 2]
```diff
@@ -774,7 +774,7 @@
   module: "Config"
   actiongroups:
      - name: "SetGroupForValidVATIdIntraUnionActionGroup"
-       description: "Goes to the 'Configuration' page for 'Customer Configuration'. Sets the 'Group For Valid VAT ID Intra Union' option. Clicks on the Save button. Validates the the Save message is present."
```
[the remark this answers is not in the document]
Comment (sanganinamrata) [reply _data/codebase/mftf/action-groups.yml]: Hi @rogyar,

Yes, I've already done it yesterday. Refer my commit: https://github.com/magento/magento2/pull/25064/files

[the remark this answers is not in the document]
Comment (dobooth) [reply _data/codebase/mftf/action-groups.yml]: Then we can remove this file from this PR.

**Pull request merged**